docs/lib/DEBUGGING.md

```diff
@@ -144,6 +144,20 @@ The OpenSDA port provides a debugger interface, which is compatible with the Seg
 
 To use the JTAG port instead, download the tools and drivers for the debugger of your choice.
 
+#### ULINK pro configuration for NXP FRDM-K64F
+
+Configure NXP FRDM-K64F according to the pdf document from arm Keil [Application Note](http://www.keil.com/appnotes/docs/apnt_287.asp).
```

Comment:
- Please change "NXP" to "the NXP".
- Please change "arm Keil" to "the Arm Keil" to match our branding rollout.
- Please lower case "application note".